Description: Upstream changes introduced in version 1.14.5-1nexenta1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fakeroot (1.14.5-1nexenta1) unstable; urgency=low
 .
   * Rebuilt for NCP4
   * Really fix dlerror() output
 .
 The person named in the Author field signed this changelog entry.
Author: Kartik Mistry <kartik.mistry@nexenta.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fakeroot-1.14.5.orig/libfakeroot.c
+++ fakeroot-1.14.5/libfakeroot.c
@@ -209,7 +209,8 @@ void load_library_symbols(void){
  /* clear dlerror() just in case dlsym() legitimately returns NULL */
     msg = dlerror();
     *(next_wrap[i].doit)=dlsym(get_libc(), next_wrap[i].name);
-    if ( (msg = dlerror()) != NULL){
+    //if ( (msg = dlerror()) != NULL){
+    if ( (next_wrap[i].doit == NULL) && (msg = dlerror()) != NULL){
       fprintf (stderr, "dlsym(%s): %s\n", next_wrap[i].name, msg);
 /*    abort ();*/
     }
